mspm0/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+add_subdirectory(source/ti/devices)
+
+if(CONFIG_HAS_MSPM0_SDK)
+  zephyr_include_directories(
+	.
+	source
+	source/ti
+	source/ti/driverlib
+	source/ti/driverlib/m0p
+	source/ti/driverlib/m0p/sysctl
+	)
+
+  zephyr_library()
+  zephyr_library_compile_definitions(${COMPILER})
+  zephyr_library_sources(
+	source/ti/driverlib/dl_uart.c
+	source/ti/driverlib/dl_common.c
+	source/ti/driverlib/dl_i2c.c
```

Comment:
We don't have i2c in the zephyr PR. Should we rather limit this CMake specific to the ones already part of zephyr PR?

Reply:
I think it'd be better to generalize to exposing the whole SDK with a glob (done below)

Reply:
NACK! See below for comment